### DISCUSSION TITLE
access to a secured external registry

### DISCUSSION THREAD

**Discussion by matgeb83** [crc-org/crc#2339] [Q&A] [answered]
CRC is unable to pull  image from a  secured external  private registry erroring out stating   "Failed to pull image unauthorized: authentication required".  are there any patches out there for external secured image registries? or how do we add certs so that the registry can be trusted?

**Comment by cfergeau** [comment, marked as answer]
I assume you have followed https://docs.openshift.com/container-platform/4.7/openshift_images/managing_images/using-image-pull-secrets.html and maybe https://github.com/code-ready/crc/wiki/Adding-a-self-signed-certificate-registry and it does not work as expected? Can you describe in more details which step you took/how far you got?
